komga-webui/src/locales/en.json

```diff
@@ -283,9 +283,9 @@
     },
     "delete_library": {
       "button_confirm": "Delete",
-      "confirm_delete": "Yes, delete the library \"{name}\"",
+      "confirm_delete": "Yes, delete the \"{name}\" library",
       "title": "Delete Library",
-      "warning_html": "The library <b>{name}</b> will be removed from this server. Your media files will not be affected. This <b>cannot</b> be undone. Continue?"
```

Comment:
This is in the form of a riddle, and lacks a comma.

Reply:
> This is in the form of a riddle

What do you mean?



> and lacks a comma

But you didn't add one?

> Should have been quoted.

What should have been?

Reply:
I understand in some places we have quotes `"` around the names, and in others we have **bold** markers. So which one should we use consistently ? As I said above, adding raw HTML everywhere brings complexity, and I would rather not do it.

Can you expand on how **bold** is not equivalent to "quoted" ?